pkg/agent/controller/egress/egress_controller.go

```diff
@@ -223,6 +276,22 @@ func (c *EgressController) Run(stopCh <-chan struct{}) {
 		return
 	}
 
+	go c.cluster.Run(stopCh)
+
+	// The Egress has been deleted but assigned IP has not been deleted,
+	// agent should delete those IPs when it starts.
```

Comment:
so agent..
Or change "," to "." in the previous line.

Reply:
```
-       // The Egress has been deleted but assigned IP has not been deleted,
-       // agent should delete those IPs when it starts.
+       // The Egress has been deleted but assigned IP has not been deleted, so agent should delete those IPs when it starts.
```